src/pages/PortForwarding.vue

```diff
@@ -3,10 +3,10 @@
 </router>
 <template>
   <PortForwarding
-    class="content"
```

Comment:
There's CSS (around line 70) that seems to depend on this class.

Reply:
That wasn't intended.